subsys/bluetooth/shell/bt.c

```diff
@@ -192,11 +201,68 @@ static bool data_cb(struct bt_data *data, void *user_data)
 	}
 }
 
+static void print_data_set(uint8_t type, uint8_t set_value_len,
+						   const uint8_t *scan_data, uint8_t scan_data_len)
+{
+	uint8_t min_value_len = MIN(set_value_len, scan_data_len);
+
+	shell_fprintf(ctx_shell, SHELL_INFO, "          Type 0x%02x: ", type);
+	for (uint8_t value_no = 0U; value_no < (scan_data_len / min_value_len); value_no++) {
+		if (value_no > 0L) {
+			shell_fprintf(ctx_shell, SHELL_INFO, ", ");
+		}
+		shell_fprintf(ctx_shell, SHELL_INFO, "0x");
+		for (uint8_t digit_no = 0U; digit_no < min_value_len; digit_no++) {
```

Comment:
Suggest to use just `i` and `j` instead of `value_no` and `digit_no` as per https://www.kernel.org/doc/html/v4.10/process/coding-style.html#naming